week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,65 @@
+let blueChanges = document.querySelector("#blueBtn");
+blueChanges.addEventListener("click", function () {
+  document.querySelector(".jumbotron").style.backgroundColor = "#588fbd";
+  document.querySelector(".jumbotron .btn").style.backgroundColor = "#ffa500";
+  document.querySelector(".jumbotron .btn-secondary").style.backgroundColor =
+    "black";
+  document.querySelector(".jumbotron .btn-secondary").style.color = "white";
+});
+
+let orangeChanges = document.querySelector("#orangeBtn");
+orangeChanges.addEventListener("click", function () {
+  document.querySelector(".jumbotron").style.backgroundColor = "#f0ad4e";
+  document.querySelector(".jumbotron .btn").style.backgroundColor = "#5751fd";
+  document.querySelector(".jumbotron .btn-secondary").style.backgroundColor =
+    "#31b0d5";
+  document.querySelector(".jumbotron .btn-secondary").style.color = "white";
+});
+
+let greenChanges = document.querySelector("#greenBtn");
+greenChanges.addEventListener("click", function () {
+  document.querySelector(".jumbotron").style.backgroundColor = "#87ca8a";
+  document.querySelector(".jumbotron .btn").style.backgroundColor = "black";
+  document.querySelector(".jumbotron .btn-secondary").style.backgroundColor =
+    "#8c9c08";
+});
```

Comment:
Nice one, it works correctly! All three event listeners make very similar changes, therefore there is some repetition in your code. Can you think of the way how you could refactor the code so there is only one function responsible for making these changes? Hint: you can pass colors as function parameters.